File: Test Automation Engineering/Programming For QA/Programming Advanced for QA/3. Strings and Text Processing/Lab/P05.DigitsLettersAndOther.cs
using System;

namespace P05.DigitsLettersandOther
{
    class Program
    {
        static void Main(string[] args)
        {
            string text = Console.ReadLine();
            string digits = null;
            string letters = null;
            string symbols = null;

            for (int i = 0; i < text.Length; i++)
            {
                char ch = text[i];
				
                if (Char.IsDigit(ch))
                {
                    digits += ch;
                }
				
                else if (Char.IsLetter(ch))
                {
                    letters += ch;
                }
				
                else
                {
                    symbols += ch;
                }
            }
			
            Console.WriteLine(digits);
            Console.WriteLine(letters);
            Console.WriteLine(symbols);
        }
    }
}

File: JAVA Fundamentals/28. Text Processing/P05_DigitsLettersAndOther.java
package textProcessing;

import java.util.Scanner;

public class P05_DigitsLettersAndOthers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String text = scanner.nextLine(); //"Agd#53Dfg^&4F53"

        StringBuilder digits = new StringBuilder();
        StringBuilder letters = new StringBuilder();
        StringBuilder others = new StringBuilder();

        for (char symbol : text.toCharArray()) {
            if (Character.isDigit(symbol)) {
            
              digits.append(symbol);
            } else if (Character.isLetter(symbol)) {
               
                letters.append(symbol);
            } else {
               
                others.append(symbol);
            }
        }

        System.out.println(digits);
        System.out.println(letters.toString());
        System.out.println(others.toString());

    }
}
